Templates/JenkinsPipeline/README.md

```diff
@@ -0,0 +1,132 @@
+  * to publish log files to the Jenkins build result.
+* `SonarQube Analysis` stage to request a SonarQube scan of the application repository.
+* `Packaging` stage
+  * to create a UrbanCode Component version via the Common Backend Script [ucdPackaging.sh](../Common-Backend-Scripts/README.md#45---ucdpackagingsh),
+  * to publish packaging log files to the Jenkins build result,
+  * to add links to the UCD Component version.
+* `Deploy to INT` stage to deploy to the development / integration test environment that includes:
+  * to request the Deployment of the UrbanCode Component version via the Common Backend Script [ucdDeploy.sh](../Common-Backend-Scripts/README.md#46---ucddeploysh) to the shared Development/Integration test environment,
+  * to publish deployment log files to the Jenkins build result,
+  * to add links to the UCD Deployment request.
+ * `Workspace Cleanup` stage to clean up the workspace.
+
+Depending on your selected and software analysis and deployment technology, review the definitions and (de-)/activate the appropriate steps.
+
+The pipeline is implemented as a declarative pipeline.
+
+## Prerequisites
+
+The [Common Backend Scripts](../Common-Backend-Scripts/) need to be configured for the selected technologies to operate correctly.
+
+## Installation and setup of template
+
+**Note: Please work with your pipeline specialist to review the below section.**
+
+The `Jenkinsfile` can be placed into the root folder of your Git repository and will automatically provide pipelines for the specified triggers defined in the Jenkins Multibranch Pipeline job configuration.
+
+Please review the definitions thoroughly with your Jenkins administrator. Ideally, the Jenkinsfile is converted into a [Jenkins Shared Library](https://www.jenkins.io/doc/book/pipeline/shared-libraries/) to provide stronger, central control.
+
+Step-by-step instructions to define and configure a Jenkins Multibranch Pipeline is documented at: https://www.jenkins.io/doc/book/pipeline/multibranch/
+
+
+## Pipeline usage
+
+The pipeline template implements the common clone, build, package and deploy steps to process various configurations according to the defined conventions. It leverages the a declarative checkout of the application's Git repository.
+
+It is supporting the following workflows: 
+
+* automated [build pipelines for feature branches](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#pipeline-build-of-feature-branches) with a clone and build stage
+* [basic pipeline](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#the-basic-build-pipeline-for-main-epic-and-release-branches) triggered when feature branches are merged into the branch `main`
+* [release pipeline](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#the-release-pipeline-with-build-packaging-and-deploy-stages) to build, package and deploy a release via UCD
```

Comment:
URL correction: https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline/#the-release-pipeline-with-build-package-and-deploy-stages